Makefile

```
@@ -58,7 +58,7 @@ $(services): target/spec $(openapi-generator-jar)
 	mv target/out/src/Adyen.Model/$@/* Adyen/Model/$@
 
 # Generate a full client (models and service classes)
-LegalEntityManagement: target/spec $(openapi-generator-jar)  
+PosTerminalManagement: target/spec $(openapi-generator-jar)  
```

We can introduce a variable here instead of replacing it for every service: 
```suggestion
full-services:=LegalEntityManagement PosTerminalManagement
$(full-services): target/spec $(openapi-generator-jar)  
```